receiver/opencensusreceiver/octrace/opencensus.go

```diff
@@ -53,6 +56,7 @@ func New(instanceName string, nextConsumer consumer.TraceConsumer, opts ...Optio
 	ocr := &Receiver{
 		nextConsumer: nextConsumer,
 		instanceName: instanceName,
+		tracer:       global.Tracer("go.opentelemetry.io/collector/receiver/opencensus"),
```

Comment:
At first it seems that the "opencensus" suffix is not consistent with the tracer for other receivers and components, ie.: they will be under "exporter", "receiver", and so on.